src/parser.js

```diff
@@ -0,0 +1,40 @@
+const bent = require('bent');
+const jsdom = require('jsdom');
+
+const request = bent('string');
+const url = 'https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List';
```

Comment:
This should get moved into our `env.sample` file and be read from `process.env` using the `src/config` module.

Reply:
I don't see these files you mentioned, is anyone still working on them?

Reply:
See the work in #145 by @hansal7014 or what @miggs125 is doing in #141.  See https://github.com/Seneca-CDOT/telescope/blob/master/env.example and https://github.com/Seneca-CDOT/telescope/blob/master/src/config.js.